client/src/pages/Bookings/useStyles.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    justifyContent: 'center',
+  },
+  upcomingBookings: {
+    '@media(min-width: 600px)': {
+      order: -1,
+    },
+    '& .MuiBox-root': {
+      boxShadow:
+        '0px 0px 1.9px rgba(0, 0, 0, 0.007),0px 0px 4.9px rgba(0, 0, 0, 0.014),0px 0px 9.9px rgba(0, 0, 0, 0.021),0px 0px 20.4px rgba(0, 0, 0, 0.031),0px 0px 56px rgba(0, 0, 0, 0.05)',
+      marginBottom: theme.spacing(2),
+      padding: theme.spacing(3),
+      textTransform: 'uppercase',
+      fontSize: '.7rem',
+      fontWeight: 'bold',
+    },
+    '& .MuiGrid-root': {
+      flexDirection: 'column',
+    },
```

Comment:
Ideally, we add these styles directly to the components instead of targeting nested elements with CSS. You can also apply some of the styles here directly to the components (e.g. both `Box` and `Grid` components accept all [system properties](https://mui.com/system/properties/)). Use your discretion to decide what styles are good to see explicitly and what's better to tuck away in order to keep the code more readable.

Reply:
Oh I did not realize that. I will certainly use those more often, as styling with CSS specificity is a bit of a pain. Thank you!